cmd/boost/direct_deal.go

```diff
@@ -134,6 +152,36 @@ var directDealAllocate = &cli.Command{
 			return fmt.Errorf("requested datacap is greater then the available datacap")
 		}
 
+		// Setup policy for allocations
+		termMin := abi.ChainEpoch(verifregst.MinimumVerifiedAllocationTerm)
+		termMax := abi.ChainEpoch(verifregst.MaximumVerifiedAllocationTerm)
+		exp := abi.ChainEpoch(verifregst.MaximumVerifiedAllocationExpiration)
+
+		if cctx.IsSet("termmin") {
+			termMin = abi.ChainEpoch(cctx.Int64("termmin"))
+		}
+
+		if cctx.IsSet("termmin") {
+			termMax = abi.ChainEpoch(cctx.Int64("termmax"))
+		}
+
+		if cctx.IsSet("termmin") {
+			exp = abi.ChainEpoch(cctx.Int64("expiration"))
+		}
```

Comment:
I'd suggest we set the Value field of the flag to the appropriate constant, eg
```Value=verifregst.MinimumVerifiedAllocationTerm```
Then we can simplify this code to
```
termMin := abi.ChainEpoch(cctx.IsSet("termmin"))
...
```